app/models/manageiq/providers/ibm_terraform/configuration_manager/configuration_profile.rb

```diff
@@ -2,7 +2,7 @@ class ManageIQ::Providers::IbmTerraform::ConfigurationManager::ConfigurationProf
   supports :native_console
 
   def console_url
-    base_url = provider.default_endpoint.url
+    base_url = provider.cpd_endpoint.url
```

Comment:
NOTE for existing providers this will be `nil` since the user will not have added the additional endpoint

Reply:
Wonder if there should be a data migration to just create the cpd endpoint based on another endpoint.

Reply:
yeah that assumes that we know the endpoint name, wonder if for now it'd be enough to add a check to `supports :native_console` and say the unsupported reason is they have to add a cpd_endpoint url

Reply:
Yeah that's an option - we can generally assume the name of the URL though from another URL, so that might be simpler

Reply:
We could do a data migration, but I guess we have the same issue with picking a service/url name to do a migration as we have with just taking the base url and building the url which is that it might be wrong if the user changed it.

Reply:
Solved by the check to see if the endpoint is present https://github.com/ManageIQ/manageiq-providers-ibm_terraform/pull/84/files#diff-6c76a5a19a1cbffd7771177da620b0437e18369e9c3f09339b2dbbd10e747bb5R3